smartcontractkit/external-adapters-js
# Add get-release-notes script to naively generate release note markdow…

## Description
Add get-release-notes script to naively generate release note markdown from .changeset files

Example:
```
➜  external-adapters-js git:(automate-release-notes) yarn get-release-notes

# Changeset
## Breaking changes (major)
- Remove synthetixjs/contracts-interface library and fetch contract addresses from AddressResolver contract
  - synthetix-debt-pool-adapter

## Features (minor)
- PRO_API_ENDPOINT added in config
  - coinpaprika-adapter

## Bug fixes (patch)
- Added buildUrl & buildUrlPath methods to util. Updated source adapters to use these methods for building URLs with user input.
  - oilpriceapi-adapter, paxos-adapter, polygon-adapter, satoshitango-adapter, sochain-adapter, sportsdataio-adapter, therundown-adapter, tiingo-adapter, tradingeconomics-adapter, uscpi-one-adapter, xbto-adapter
- Add global handler for unhandled promise rejections
  - ea-bootstrap

|    Adapter    | Version |
| :-----------: | :-----: |
| @chainlink/coinpaprika-adapter | 1.3.11 |
| @chainlink/synthetix-debt-pool-adapter | 2.1.0 |
| @chainlink/oilpriceapi-adapter | 2.0.20 |
| @chainlink/paxos-adapter | 1.2.20 |
| @chainlink/polygon-adapter | 1.4.18 |
| @chainlink/satoshitango-adapter | 1.2.20 |
| @chainlink/sochain-adapter | 1.2.20 |
| @chainlink/sportsdataio-adapter | 1.1.34 |
| @chainlink/therundown-adapter | 1.2.20 |
| @chainlink/tiingo-adapter | 1.8.0 |
| @chainlink/tradingeconomics-adapter | 1.1.20 |
| @chainlink/uscpi-one-adapter | 1.1.20 |
| @chainlink/xbto-adapter | 1.2.20 |
| @chainlink/ea-bootstrap | 1.13.0 |
```

## Changes
- Add script
- Add GitHub action step to run prior to consuming `.changset` files

## Steps to Test

1. Run `yarn get-release-notes` in the external-adapters-js repo
2. Observe that release notes are generated

## Quality Assurance
